### ISSUE TITLE
Brave news sources are incorrectly alphabetized

### ISSUE THREAD

**Issue by bridiver** [brave/brave-browser#19365]
![image](https://user-images.githubusercontent.com/34129/141213856-f522f204-8211-45cc-8d56-e3fbdee07fac.png)

"When alphabetizing titles or group names as authors, go by the first significant word (disregard a, an, the, etc.)" - https://blog.apastyle.org/apastyle/2010/05/alphabetization-in-apa-style.html and other sources

**Comment by mattmcalister** [comment]
Closing as dupe https://github.com/brave/brave-browser/issues/19731

**Comment by bridiver** [comment]
@mattmcalister I don't think this is a dupe of that issue, this issue is about incorrectly alphabetizing based on "The" or other articles instead of the first non-article word

**Comment by bridiver** [comment]
this is definitely not a dupe, this is about the individual feeds being alphabetized incorrectly, #19731 is about the list of source categories
